content/en/docs/reference/glossary/ResourceClaimParameters.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ResourceClaimParameters
+id: ResourceClaimParameters
+date: 2023-10-16
+full_link: /docs/concepts/scheduling-eviction/dynamic-resource-allocation/
+short_description: >
+  Specification of what and how much of resources the ResourceClaim needs.
+aka:
+tags:
+- extension
+---
+ {{< glossary_tooltip term_id="resource-driver" text="Resource Driver">}}-specific object, subject
+to vendor implementation. Optional. Typically contains quantity and characteristics of the requested
+resources.
+
+<!--more-->
+
+Not part of core Kubernetes. Referenced in `ParametersRef` field of
```

Comment:
```suggestion
```
Instead, I suggest we link to the page about DRA for readers to learn more details.